### PR TITLE
[CSL-1436] Rename travis mode to ci mode, use in appveyor

### PR DESCRIPTION
**Pull request by gromakovsky**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,7 +26,7 @@ cache:
 before_install:
 - echo $TRAVIS_OS_NAME
 - export CSL_SYSTEM_TAG=$TRAVIS_OS_NAME
-- export DCONFIG=travis
+- export DCONFIG=ci
```
Comment (domenkozar) [inline .travis.yml]: Shouldn't this also be `ci_wallet`?

Comment (gromakovsky) [inline .travis.yml]: Yes, I think it should, I will fix, thanks.

**Comment by gromakovsky** [comment]
Ok, then I will disable `VarSpec` in appveyor until we resolve it.

**Comment by domenkozar** [comment]
We should at least use the same constants on appveyor/travis, it's really confusing to track these.

**Comment by gromakovsky** [comment]
Okay, another workaround: https://github.com/input-output-hk/cardano-sl/pull/1232

**Comment by gromakovsky** [comment]
Closed, but see #1235.

**Pull request closed without merging**